shared/js/background/chrome-events.es6.js

```diff
@@ -426,59 +439,18 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
 // TODO fix for manifest v3
 let sessionKey = getHash()
 
-async function getContentScope () {
-    const url = chrome.runtime.getURL('/public/js/content-scope.js')
-
-    const response = await fetch(url)
-    return response.text()
-}
-
-async function init () {
-    const contentScopeScript = await getContentScope()
-
-    // Inject fingerprint protection into sites when
-    // they are not whitelisted.
-    chrome.webNavigation.onCommitted.addListener(details => {
-        const tab = tabManager.get({ tabId: details.tabId })
-        if (tab && tab.site.isBroken) {
-            console.log('temporarily skip fingerprint protection for site: ' + details.url +
-            'more info: https://github.com/duckduckgo/content-blocking-whitelist')
-            return
-        }
-        if (tab && !tab.site.whitelisted) {
-            // Set variables, which are used in the fingerprint-protection script.
-            try {
-                const argumentsObject = {
-                    stringExemptionLists: utils.getBrokenScriptLists(),
-                    sessionKey,
-                    contentScopeScript,
-                    site: tab.site,
-                    referrer: tab.referrer
-                }
-                const variableScript = {
-                    code: `
-                      try {
-                          var ddg_args = ${JSON.stringify(argumentsObject)}
-                      } catch(e) {}`,
-                    runAt: 'document_start',
-                    frameId: details.frameId,
-                    matchAboutBlank: true
-                }
-                chrome.tabs.executeScript(details.tabId, variableScript)
-                const scriptDetails = {
-                    file: '/public/js/injected-content-scripts/fingerprint-protection.js',
-                    runAt: 'document_start',
-                    frameId: details.frameId,
-                    matchAboutBlank: true
-                }
-                chrome.tabs.executeScript(details.tabId, scriptDetails)
-            } catch (e) {
-                console.log(`Failed to inject fingerprint protection into ${details.url}: ${e}`)
-            }
-        }
-    })
+function getArgumentsObject (tabId) {
+    const tab = tabManager.get({ tabId })
+    const site = tab?.site || {}
```

Comment:
So if the tabManager has no info on the tab, protections are on?

Reply:
I've changed this back to how it was. I'm not sure if there is rationale in either way... perhaps it solves a race condition.